src/game/_main.cpp

```diff
@@ -102,11 +142,17 @@ main(int argc, char* argv[])
     float relative_tilemap_mouse_x;
     float relative_tilemap_mouse_y;
 
+    ImVec2 panel_size = { (float)(auto_tile_map.tile_size * auto_tile_map.n_cols), (float)(auto_tile_map.tile_size * auto_tile_map.n_rows)};
+    ImVec2 screen_pos;
+    ImGuiID dockspace_id = NULL;
```

Comment:
ImGuiId is not a pointer type, so i don't think it should be initialized with NULL/nullptr values.